controllers/user-controllers.js

```diff
@@ -42,3 +47,15 @@ export const currentUser = async (req, res) => {
   const { email } = req.body;
   res.status(200).json({ email });
 };
+
+export const updateAvatar = async (req, res) => {
+  const { _id } = req.user;
+  const { path: tmpPath, filename } = req.file;
+  const newPath = path.join(__dirname, "..", "public", "avatars", filename);
```

Comment:
można też napisać to krócej te w środku `../public/avatars/`
Ale tak też jest spoko :)